src/containers/shared/components/Transaction/OfferCreate/OfferCreateDescription.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { useTranslation, Trans } from 'react-i18next';
+import { localizeDate } from '../../../utils';
+import {
+  DATE_OPTIONS,
+  CURRENCY_ORDER,
+  RIPPLE_EPOCH,
+  XRP_BASE,
+  normalizeAmount,
+} from '../../../transactionUtils';
+import Account from '../../Account';
+import { TransactionDescriptionComponent, TransactionDescriptionProps } from '../types';
+
+const normalize = (amount: any) => amount.value || amount / XRP_BASE;
+
+const OfferCreateDescription: TransactionDescriptionComponent = (
+  props: TransactionDescriptionProps
+) => {
+  const { t, i18n } = useTranslation();
+  const language = i18n.resolvedLanguage;
+  const { data } = props;
+  const paysCurrency = data.tx.TakerPays.currency || 'XRP';
```

Comment:
if I'm reading this correctly, I can just directly access the tx data without it needing to go through the instructions, right?

Reply:
I will have to look into this. 90% of this is code that already existed.

Reply:
For some reason the description takes the whole transaction.  I made it that way because that is how its been.